static/js/compose.js

```diff
@@ -686,6 +690,31 @@ export function initialize() {
         }
     });
 
+    $("body").on("click", ".add_poll", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        dialog_widget.launch({
+            html_heading: $t_html({defaultMessage: "Add poll"}),
+            html_body: render_add_poll_modal(),
+            html_submit_button: $t_html({defaultMessage: "Send poll"}),
+            on_click: (e) => {
+                // create a message using data input in modal, then send that directly
+                e.preventDefault();
+                e.stopPropagation();
+                const poll_message_content = poll_modal.frame_poll_message_content();
+                const poll_message_object = create_message_object(poll_message_content);
+                const clear_compose_box_after_sending = false;
+                send_message(poll_message_object, clear_compose_box_after_sending);
+            },
+            close_on_submit: true,
+            form_id: "add-poll-form",
+            id: "add-poll-modal",
+            post_render: poll_modal.poll_options_setup,
```

Comment:
One option is to define an anonymous function that does the `setTimeout` you need here, with a comment, if we decide changing the `post_render` semantics doesn't make sense.

Reply:
The simplebar issue has been resolved as explained in this [CZO thread](https://chat.zulip.org/#narrow/stream/49-development-help/topic/add.20poll.20modal.20acting.20buggy)